### DISCUSSION TITLE
Getting some errors in a nextjs 14 mono repo

### DISCUSSION THREAD

**Discussion by fyzz-dev** [shellscape/jsx-email#186] [Q&A]
TypeError: Cannot convert a Symbol value to a string. Using the base template for testing just renamed and removed props.
```ts
// route.tsx
import { render } from "jsx-email";

import { sendEmail, Verify } from "@undrground/email";

export async function POST(_req: Request) {
  const html = await render(<Verify code="123456" />);
  await sendEmail({
    to: ["some email"],
    from: "some email",
    subject: "Verify",
    html,
  });

  return new Response("Hello world!");
}
```

**Comment by shellscape** [comment]
It's probably an incompatibility between the other package you're using. We'll need a reproduction to triage this for you. You can create one by using https://stackblitz.com/fork/jsx-email-repro
